templates/body.html

```diff
@@ -0,0 +1,6 @@
+<html>
+    <body>
```

Comment:
Te falta agregar la metadata con el charset para que peudas renderizar símbolos (Ej: `º`) y para que se adapte al ancho de la pantalla del dispositivo.
```suggestion
    <head>
      <meta name="viewport" content="width=device-width"/>
      <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    </head>
    <body>
```